cpu/cortexm_common/panic.c

```diff
@@ -32,3 +40,14 @@ void panic_arch(void)
     }
 #endif
 }
+
+#ifdef DEVELHELP
+static void print_IPSR(void)
+{
+    uint8_t ipsr = (uint8_t)(__get_IPSR() & 0xFF);
```

Comment:
Please replace `& 0xFF` by `& IPSR_ISR_Msk` to avoid magic numbers, and auto fix for later Cortex-M processors. :)